jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -143,10 +146,12 @@ public void testDockerDaemon_empty() throws IOException, InterruptedException {
 
   @Test
   public void testDockerDaemon_simple() throws IOException, InterruptedException {
+    Instant beforeBuild = Instant.now();
```

Comment:
We might want to keep one of them using the default 0, perhaps the empty one?

Reply:
Empty one still has the default 0, but I can add checks for it.